test/plugins/nest.spec.js

```diff
@@ -121,14 +139,58 @@ describe('Plugin', () => {
             done()
           }, 4) // run when 4 spans are received by the agent
 
-          app.listen(port, 'localhost')
-            .then((done) => {
-              axios
-                .get(`http://localhost:${port}/users`)
-                .catch(done)
-            })
+          axios
+            .get(`http://localhost:${port}/users`)
             .catch(done)
         }).timeout(5000)
+
+        it('should properly record errors', done => {
```

Comment:
This does not yet test for errors but modifies the test case in a way that allows multiple unit tests for nest. Earlier one server was being started per test. The tests for error handling will be added in a different PR.